internal/js/runner.go

```diff
@@ -360,7 +362,7 @@ func (r *Runner) HandleSummary(ctx context.Context, summary *lib.Summary) (map[s
 		}
 	}()
 
-	summaryCtx, cancel := context.WithTimeout(ctx, r.getTimeoutFor(consts.HandleSummaryFn))
+	summaryCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
```

Comment:
Out of curriocity, why do we switch to a 10 mins timeout? :thinking:

Reply:
Likely something I modified while debugging, that I forgot to set back, so great catch! 👍

Reply:
Done @ https://github.com/grafana/k6/pull/4089/commits/58e3d021ea7509bfa7aba35b0babefb7c54e04cc